app/src/lib/stores/app-store.ts

```diff
@@ -339,8 +339,9 @@ export class AppStore extends TypedBaseStore<IAppState> {
     this.accountsStore.onDidError(error => this.emitError(error))
 
     this.repositoriesStore.onDidUpdate(async () => {
-      const repositories = await this.repositoriesStore.getAll()
-      this.repositories = repositories
+      this.repositories = await this.repositoriesStore
+        .getAll()
+        .then(repos => this.getRefreshedRepositories(repos))
```

Comment:
Just curious about the decision to mix the `Promise.then` API with `await`. It should be possible to do the following since calling `await` on a Promise resolves it.

```ts
...
const repositories = await this.repositoriesStore.getAll()
this.getRefreshedRepositories(repositories)
this.repositories = repositories
...
```

Reply:
To me, it's more clear this way that we are waiting for the whole chain of actions to complete before assigning the results to `this.repositories`. But I don't feel strongly about this at all, so I am ok with changing this for consistency.

Reply:
fixed!